interface-elements-for-web/articles/diagram/first-look-at-the-diagram.md

```diff
@@ -0,0 +1,56 @@
+---
+title: First Look at the Diagram
+author: Svetlana Nikulina
+---
+# First Look at the Diagram
+
+![Tools](../../images/diagram-tools.png)
+
+## Diagram Tools
+
+- The **History** toolbar allows you to undo and redo your actions. On mobile devises the toolbar additionally displays the **Show Toolbox** button that shows/hides the **Toolbox**.
+
+    ![Show Toolbox Button](../../images/diagram-history-and-toolbox-button.png)
+
+- The **Toolbox** contains shapes available in the **Diagram**. Select the **View** > **Settings** > **Show Toolbox** command to switch the **Toolbox** visibility. On mobile devises the **Show Toolbox** button is additionally displayed in the **History** toolbar.
+
+- The **Properties** toolbar allows you to [customize shape and connector style settings](create-a-new-chart.md#customize-shape-and-connector-style-settings), change the [chart layout](chart-layout.md), and specify [page settings](page-and-view-settings.md). Click the **Properties** button to show and hide the toolbar.
+
+- The **View** toolbar contains commands related to **Diagram**'s [view settings](page-and-view-settings.md).
+
+- The **Context Toolbox** appears when you draw a connector from a shape and release it without it being connected to another shape. Click a shape to create it at the end of the connector. Note, this toolbox can contain a reduced set of shapes. See the **Toolbox** for the full list of available shapes.

```

Comment:
```suggestion
- The **Context Toolbox** appears when you draw a connector from a shape and release it without connecting it to another shape. Click a shape to create it at the end of the connector. This toolbox can contain a reduced set of shapes. See the **Toolbox** for the full list of available shapes.

```

Reply:
Why do you add "Note, this toolbox can contain a reduced set of shapes. See the **Toolbox** for the full list of available shapes"?